### ISSUE TITLE
trying to get episodes for a show throws error

### ISSUE THREAD

**Issue by jamgold** [worr/node-imdb-api#63]
Hi, thanks for your work on this. Following your example on how to get the episodes for a show, it always fails with `TypeError: invalid release date` or `TypeError: invalid rating`, depending on what show I query.

Here is the code I use
```
  const imdbid = 'tt4270492';
  imdb.get(
    {id: imdbid},{
      apiKey: MyApiKey
  }).then((show) => {
    return show.episodes();
  }).then((episodes) => {
    console.log(episodes);
  }).catch(console.log);
```

**Comment by jamgold** [comment]
When I lookup "How I Met Your Mother" the code works as advertised, but for anything else it fails with the above error.

**Comment by worr** [comment]
Thanks for the report! Looks like there's some inconsistent data in omdb. I'll work on a patch to be more tolerant with the conversions once I'm back from vacation.

**Comment by dheffx** [comment]
i ran into a similar issue on "invalid release date". i fixed it on my end, then saw this issue so i handled that scenario as well and figured i would share. i sent a pull request if you are interested https://github.com/worr/node-imdb-api/pull/65

**Comment by worr** [comment]
This was fixed with #65
